### DISCUSSION TITLE
[v5]Running Migrations At Heroku is throwing Error: AdonisJS requires "@adonisjs/assembler"

### DISCUSSION THREAD

**Discussion by mvpxtech** [adonisjs/core#1906] [Help]
**Procfile:**
_`release: ENV_SILENT=true node ace migration:run --force
web: ENV_SILENT=true node build/server.js`_

**I also did a manual migration using:**
`heroku run -a my-app node ace migration:run --force`

**The error is:**
`AdonisJS requires "@adonisjs/assembler`

**Comment by thetutlage** [comment]
You should be running migrations from the build folder too

```
release: ENV_SILENT=true node build/ace migration:run --force web: ENV_SILENT=true node build/server.js
```
